app/src/pages/admin/serverSettingsPage/serverSettingsTabs/authConfigurationTab/forms/githubAuthForm/githubAuthFormFields/githubAuthFormFields.jsx

```diff
@@ -0,0 +1,150 @@
+import IconDelete from 'common/img/circle-cross-icon-inline.svg';
+import { FormField } from 'components/fields/formField';
+import { FieldErrorHint } from 'components/fields/fieldErrorHint';
+import { Input } from 'components/inputs/input';
+import { GhostButton } from 'components/buttons/ghostButton';
+import { showModalAction } from 'controllers/modal';
+import PlusIcon from 'common/img/plus-button-inline.svg';
+import {
+  GITHUB_AUTH_FORM,
+  CLIENT_ID_KEY,
+  CLIENT_SECRET_KEY,
+  ORGANIZATIONS_KEY,
+  NEW_ORGANIZATION_KEY,
+} from '../constants';
+import styles from './githubAuthFormFields.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  clientIdLabel: {
+    id: 'GithubAuthFormFields.clientIdLabel',
+    defaultMessage: 'Client ID',
+  },
+  clientSecretLabel: {
+    id: 'GithubAuthFormFields.clientSecretLabel',
+    defaultMessage: 'Client secret',
+  },
+  addOrganizationButtonTitle: {
+    id: 'GithubAuthFormFields.addOrganizationButtonTitle',
+    defaultMessage: 'Add GitHub organization',
+  },
+  organizationNameLabel: {
+    id: 'GithubAuthFormFields.organizationNameLabel',
+    defaultMessage: 'Organization name',
+  },
+});
+
+@connect(
+  (state) => ({
+    organizations: (formValueSelector(GITHUB_AUTH_FORM)(state, ORGANIZATIONS_KEY) || '').split(','),
+  }),
+  { showModalAction },
+)
+@injectIntl
+export class GithubAuthFormFields extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    organizations: PropTypes.array,
+    showModalAction: PropTypes.func,
+  };
+
+  static defaultProps = {
+    organizations: [],
+    showModalAction: () => {},
+  };
+
+  state = {
+    isAddOrganizationClicked: false,
+  };
+
+  addOrganization = () => {
+    this.setState({
+      isAddOrganizationClicked: true,
+    });
+  };
+
+  removeOrganizationHandler = (organizationForRemove) => {
+    this.props.showModalAction({ id: 'removeOrganizationModal', data: { organizationForRemove } });
+  };
+
+  render() {
+    const {
+      intl: { formatMessage },
+      organizations,
+    } = this.props;
+
+    return (
+      <div className={cx('github-auth-form-fields')}>
+        <FormField
+          name={CLIENT_ID_KEY}
+          required
+          fieldWrapperClassName={cx('form-field-wrapper')}
+          label={formatMessage(messages.clientIdLabel)}
+          labelClassName={cx('label')}
+        >
+          <FieldErrorHint>
+            <Input mobileDisabled />
+          </FieldErrorHint>
+        </FormField>
+        <FormField
+          name={CLIENT_SECRET_KEY}
+          required
+          fieldWrapperClassName={cx('form-field-wrapper')}
+          label={formatMessage(messages.clientSecretLabel)}
+          labelClassName={cx('label')}
+        >
+          <FieldErrorHint>
+            <Input mobileDisabled />
+          </FieldErrorHint>
+        </FormField>
+        {!!organizations.length &&
+          organizations[0] &&
```

Comment:
Yes, we can get `[""]` form `props`

Reply:
I think it will be more clear if you write it like this:
`const filteredOrganizations = organizations.filter(Boolean)`
At least this will help to understand that there may be values that we don't need to render.

Reply:
Fixed